.aicoe-ci.yaml

```diff
@@ -1,8 +1,7 @@
 check:
-  - thoth-precommit
   - thoth-build
 build:
-  base-image: "quay.io/thoth-station/s2i-thoth-f32-py38-cloc:latest"
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.35.0
```

Comment:
please revert this change 
ubi doesn't have cloc base packages.

if this package can be installed in ubi
https://github.com/thoth-station/si-cloc/blob/cea233837f6209878f92ccc51270ccba2f9da13f/Dockerfile#L4
then please lets update the Dockerfile.

Reply:
So what's the preferred option, installing cloc in the Dockerfile or using the cloc base image ?
@goern @codificat

Reply:
well... I suggested the image update when I saw "f32" in the current one. It is 2 years old and it will only keep getting older. So, not an ideal option IMHO.

But right: there's no `cloc` in the UBI repos, nor in EPEL... so installing it in UBI is probably bound to be not ideal either?

As a quick solution, maybe for the time being we can stick to the current image and accept its aging?